### DISCUSSION TITLE
What counts as a GP addon? Can I submit an addon somehow?

### DISCUSSION THREAD

**Discussion by aleksilassila** [GriefPrevention/GriefPrevention#1268] [Q&A]
I have this project of mine which uses GP to create personal islands for players to build on, so the plugin basically manages gp claims for the players. (Yep, its a open source recreation of BigScary's islands plugin, you might be familiar with the concept)

Anyways, I was wondering if the plugin meets the requirements of an addon. You can check the project [here](https://github.com/aleksilassila/Islands).

Cheers!

**Comment by RoboMWM** [comment]
Anything that hooks into GP is more or less an "addon." I'll soon setup a new page for addons in the new docs page, docs.griefprevention.com.
